packages/assets-controllers/src/selectors/token-selectors.ts

```diff
@@ -438,14 +459,66 @@ const selectAllAssets = createAssetListSelector(
   },
 );
 
+export type SelectAccountGroupAssetOpts = {
+  filterTronStakedTokens: boolean;
+};
+
+const defaultSelectAccountGroupAssetOpts: SelectAccountGroupAssetOpts = {
+  filterTronStakedTokens: true,
+};
+
+const filterTronStakedTokens = (assetsByAccountGroup: AccountGroupAssets) => {
+  const newAssetsByAccountGroup = { ...assetsByAccountGroup };
+
+  Object.values(TrxScope).forEach((tronChainId) => {
+    if (!newAssetsByAccountGroup[tronChainId]) {
+      return;
+    }
+
+    newAssetsByAccountGroup[tronChainId] = newAssetsByAccountGroup[
+      tronChainId
+    ].filter((asset: Asset) => {
+      if (
+        asset.chainId.startsWith('tron:') &&
+        TRON_RESOURCE_SYMBOLS_SET.has(
+          asset.symbol?.toLowerCase() as TronResourceSymbol,
+        )
+      ) {
+        return false;
+      }
+      return true;
+    });
+  });
+
+  return newAssetsByAccountGroup;
+};
+
 export const selectAssetsBySelectedAccountGroup = createAssetListSelector(
-  [selectAllAssets, (state) => state.accountTree],
-  (groupAssets, accountTree) => {
+  [
+    selectAllAssets,
+    (state) => state.accountTree,
+    (
+      _state,
+      opts: SelectAccountGroupAssetOpts = defaultSelectAccountGroupAssetOpts,
```

Comment:
Ok for now, but I'm not a big fan of this as it can force recalculations based on filters. Also, if we add more chains that have different needs we are going to end up with very messy logic here.

IMO, the problem with Tron should have been solved by expanding the asset type to contain underlying metada (the tron resources), so all these Tron resources would have been added inside the main Tron asset, which will always be present. By adding those "resources" to the list of assets, even though we don't want to display them most of the time, we are causing situations like this one.